docs/python/environments.md

```diff
@@ -48,16 +51,16 @@ The extension automatically looks for interpreters in the following locations:
 - Virtual environments located in a `~/.virtualenvs` folder for [virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/).
 - Interpreters installed by [pyenv](https://github.com/pyenv/pyenv), [Pipenv](https://pypi.org/project/pipenv/), and [Poetry](https://poetry.eustace.io/).
 - Virtual environments located in the path identified by `WORKON_HOME` (as used by [virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/)).
-- Conda environments that contain a Python interpreter. VS Code does not show conda environments that don't contain an interpreter.
+- Conda environments that contain a Python interpreter. VS Code doesn't show conda environments that don't contain an interpreter.
```

Comment:
I believe this will change in the next release, right @karrtikr? We now install a Python interpreter on conda environments that don't come with one?

Reply:
Correct.
```suggestion
- Conda environments found by `conda env list`. Conda environments which do not have an interpreter will have one installed for them upon selection.
```